packages/common-react/src/toastify-components/stlite-kernel-with-toast.tsx

```diff
@@ -10,9 +10,9 @@ function stliteStyledPromiseToast<
 >(
   promise: Promise<TData>,
   messages: ToastPromiseParams<TData, TError, TPending>,
-): ReturnType<typeof toast.promise> {
+): Promise<TData> {
   const errorMessage = messages.error;
-  return toast.promise<TData, TError, TPending>(
+  toast.promise<TData, TError, TPending>(
```

Comment:
Not necessary change, or worse?
Do you understand the `toast.promise`'s API? The returned promise from `toast.promise` is floating here.